File: app/src/main/cpp/include/lib_log/LogUtil.cpp
//
// Created by glumes on 2017/5/26.
//

#include "LogUtil.h"
#include <string>
#include <android/log.h>
#define TAG "NativeCodec-looper"
#define LOGD(...) __android_log_print(ANDROID_LOG_DEBUG, TAG, __VA_ARGS__)

using namespace std;

LogUtil::LogUtil() {

}

void LogUtil::test(string msg) {
    LOGD("this is content two");
}

File: cppso/src/main/java/com/glumes/cppso/utils/LogUtil.kt
package com.glumes.cppso.utils

import android.util.Log

/**
 * Created by glumes on 28/02/2018
 */
class LogUtil {

    companion object {

        var TAG: String = "NativeMethod"

        fun d(msg: String) {
            d(msg, tag = TAG)
        }

        fun i(msg: String) {
            i(msg, tag = TAG)
        }

        fun d(msg: String, tag: String = TAG) {
            Log.d(tag, msg)
        }

        fun i(msg: String, tag: String = TAG) {
            Log.i(tag, msg)
        }
    }
}